cobbler/items/profile.py

```diff
@@ -457,10 +457,19 @@ def filename(self, filename: str):
         """
         if not isinstance(filename, str):
             raise TypeError("Field filename of object profile needs to be of type str!")
+        parent = self.parent
+        if (
+            filename == enums.VALUE_INHERITED
+            and parent
+            and parent.TYPE_NAME == "distro"
+        ):
+            raise ValueError('Property "filename" cannot inherit from distro!')
```

Comment:
Not sure here, but looks like the meaning of the error is bit different
```suggestion
            raise ValueError('Property "filename" cannot be inherited from distro!')
```

Reply:
So if I translate that to German, in this specific context I believe it doesn't make a difference. Maybe @agraul has some insight? I am very bad with such things...

Reply:
I'm no language expert in any way, shape or form. To me the passive tone sounds better in this case, based on the thoughts below.

Both versions leave something out:
- active: what's the object that the "filename" property can't inherit?
- passive: what subject tries to inherit the "filename" property?

Then the question is "which omission is less of a problem" and I'd say the subject in the passive version.

Reply:
Since I made the code more forgiving, this is not an issue anymore.